packages/core/src/config/utils.ts

```diff
@@ -157,6 +157,9 @@ export function getMergedConfig(config: Config, apiName?: string): Config {
             ...config['features.mockServer'],
             ...config.apis[apiName]?.['features.mockServer'],
           },
+          files: config.apis[apiName]
+            ? config.files.concat(config.apis[apiName].files ?? [])
+            : config.files,
```

Comment:
I believe we don't need this ternary: 
```suggestion
          files: [...config.files, ...config.apis?.[apiName]?.files ?? []],
```